pkg/kubecost/window.go

```diff
@@ -432,6 +444,86 @@ func (w Window) Minutes() float64 {
 	return w.end.Sub(*w.start).Minutes()
 }
 
+// Overlaps returns true iff the two given Windows share and amount of temporal
+// coverage.
+// TODO complete (with unit tests!) and then implement in AllocationSet.accumulate
```

Comment:
It's fine if we launch without this. It's from a never-used use case where we would throw an error if someone tries to accumulate sets with overlapping windows (e.g. `accumulate(today, last 24h)`) which, again, never happens.